build/autoloaderchecker.sh

```diff
@@ -22,6 +22,12 @@ echo "Regenerating main autoloader"
 $COMPOSER_COMMAND dump-autoload -d $REPODIR
 
 for app in ${REPODIR}/apps/*; do
+	git check-ignore ${app} -q
+	if git check-ignore ${app} -q ; then
```

Comment:
Why call the command twice?
```suggestion
	if git check-ignore ${app} -q ; then
```